media/com_redshop/js/redshop.common.js

```diff
@@ -1054,11 +1054,16 @@ function onestepCheckoutProcess(objectname, classname, anonymous)
 
 	if (jQuery('.extrafield_payment').length)
 	{
-		jQuery('.extrafield_payment').children('[id^="extraFields"]').remove();
+		jQuery('#paymentblock .payment_method').each(function () {
+			if (!jQuery(this).is(':checked')) {
+				jQuery(this).find('#extraFields_rs_payment_banktransfer2').remove();
```

Comment:
please double check if we need show "bank transfer" again or not

Reply:
I checked, if remove code it will display custom fields of payment banktransfer2.